pkg/scheduler/framework/runtime/framework.go

```diff
@@ -835,6 +892,9 @@ func (f *frameworkImpl) RunFilterPluginsWithNominatedPods(ctx context.Context, s
 	// the nominated pods are treated as not running. We can't just assume the
 	// nominated pods are running because they are not running right now and in fact,
 	// they may end up getting scheduled to a different node.
+	logger := klog.FromContext(ctx)
+	logger = klog.LoggerWithName(logger, "WithNominatedPods")
```

Comment:
About the name, anyone has better options?

Reply:
We can be more explicit

```suggestion
	logger = klog.LoggerWithName(logger, "FilterWithNominatedPods")
```

Reply:
Done.